src/services/UserProfileApiService.js

```diff
@@ -0,0 +1,65 @@
+import camelcaseKeys from 'camelcase-keys';
```

Comment:
Ideally, we would keep these functions together with the read functions to create a comprehensive API service layer for the user account API. The best place for that service layer is probably in a separate npm package and not `frontend-auth`. In order of preference, I think we either want to do that refactoring now, leave the write functions in frontend-auth for the time being, or leave the write functions here and remove them from frontend-auth.

Reply:
I'm assuming that in the long run there's going to be a difference between APIs that are used to get user session data and APIs that are used to manage profile data for an arbitrary user.  In that sense - and because some of the refactoring called for pulling them out - it felt like the methods to save a user profile don't belong in a frontend-auth package.  

That said, it's also not clear to me that having all API methods available in a comprehensive package is going to be the right thing to do either.  Given those three possibilities, it felt most correct for them to live in the profile app, so I put them here as part of pulling the reducer and other profile-page specific code out of frontend-auth. 

We could refactor that further, but my understanding is that for v0.x we're not concerning ourselves with broader refactorings like that.

Reply:
I think in the end it will make sense for fetch functions to live with save functions at the API service layer. If we don't want to do option #1, can we at least go with option #2 (leave the save functions in frontend-auth)?

Reply:
I guess I'm not quite clear why profile saving functions would go in an "auth" repository... is frontend-auth actually more of a frontend-api package?

Reply:
I mean, I'm all for having them all in one place!  But it just seems like a misnomer, kinda.

Reply:
I totally agree that `frontend-auth` is not the right place for the `UserAccountApiService`, hence my #1 option above (refactor it to a separate `frontend-service-user-account` or whatever name npm package). I think it's better to keep everything together until we want to to that refactor and everything is already together in `frontend-auth`. For added context, the fetch interface is currently in use by other apps that consume `frontend-auth`, namely https://github.com/edx/edx-portal.